### PR TITLE
Issue #CO-486

### PR DESCRIPTION
**Pull request by rekhaLuezoid**
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

### Type of change

Please choose appropriate options.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### How Has This Been Tested?

Please describe the tests that you ran to verify your changes in the below checkboxes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Ran Test A
- [ ] Ran Test B

**Test Configuration**:
* Software versions: Nodejs 8.11.2
* Hardware versions:

### Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by vinukumar-vs** [comment]
@vaivk369 Can you review the PR.

File: src/service/print/csv.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 const { getQuestionSet } = require("./printDocxV1.0/dataImporter");
-//const JSON2CSV = require("@json2csv/node").parse;
+let JSON2CSV = require('json-2-csv');
```
Comment (vaivk369) [inline src/service/print/csv.js:2]: @rekhaLuezoid package should be added as dependency in package.json

File: src/package.json [hunk 2]
```diff
@@ -37,6 +37,7 @@
     "http-errors": "1.6.3",
     "image-size": "0.9.7",
     "jade": "1.11.0",
+    "json-2-csv": "^4.0.0",
```
Comment (vaivk369) [inline src/package.json:40]: @rekhaLuezoid please remove @json2csv/node from the list as it is not being used now.. Also, remove the '^' from the version. It should be  "json-2-csv": "4.0.0",

File: src/service/print/csv.js [hunk 3]
```diff
@@ -126,13 +126,10 @@ const buildCSVWithCallback = async (id, callback) => {
           "LeftColumn",
         ];
 
-        // let csv = JSON2CSV(questionPaperContent, {
-        //   fields: fields,
-        //   withBOM: true,
-        // });
+        let csv = await JSON2CSV.json2csv(questionPaperContent);
```
Comment (vaivk369) [inline src/service/print/csv.js:129]: @rekhaLuezoid remove fields variable as it not being used now

**Review by vaivk369 — APPROVED** [review]

**Review by vinukumar-vs — APPROVED** [review]

**Pull request merged**